specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5034,6 +5034,7 @@
         },
         "userAssignedIdentities": {
           "type": "object",
+          "x-ms-client-name": "IdentityUserAssignedIdentitiesValue",
```

Comment:
what is this for?

Reply:
As mentioned in description, it's an autorest team suggestion to avoid a v3 code generation issue: [autorest.dontnet-v3 generates random name: Componentssgqdofschemasidentitypropertiesuserassignedidentitiesadditionalproperties for userAssignedIdentities](https://github.com/Azure/autorest.csharp/issues/614)